autorecsys/pipeline/preprocessor.py

```diff
@@ -105,37 +105,207 @@ def preprocessing(self, **kwargs):
 
 class AvazuPreprocessor(BaseCTRPreprocessor):
 
-    def __init__(self, dataset_path, save_path):
+    def __init__(self, dataset_path="./examples/datasets/avazu/sampled_train_10000.txt"):
         super(AvazuPreprocessor, self).__init__(dataset_path=dataset_path, )
-        self.save_path = save_path
-        self.categ_filter = 10
+        # /home/thwang1231/autorec/examples/datasets/avazu/sampled_train_10000.txt  # 5/4 loss: 0.6239
+        # /home/thwang1231/autorec/examples/datasets/avazu/sampled_train_2mil.txt   # 5/4 loss: 0.3718
+        # /home/thwang1231/autorec/examples/datasets/avazu/train                    # 5/4 loss: 0.3859
+        # Step 1: Set attributes used/set during dataset loading & data preprocessing.
+        self.has_header = True
+        self.column_names = None
+        self.fit_dictionary_path = "./avazu_fit_dictionary.pkl"
+        self.transformed_data_path = "./avazu_transformed_data.pkl"
+        self.categ_filter = 3
+
+        # TODO (for printed message): Notice typo "BinaryCrossentropy" should be "BinaryCrossEntropy"
```

Comment:
tf.keras.metric use BinaryCrossentropy

Reply:
Yes the class is named BinaryCrossentropy in tf.keras.metric. Hence it is not a typo